src/Illuminate/Database/Query/Grammars/MySqlGrammar.php

```diff
@@ -524,4 +524,19 @@ protected function wrapJsonBooleanSelector($value)
 
         return 'json_extract('.$field.$path.')';
     }
+
+    /**
+     * Apply custom ordering to a query based on a priority array.
+     *
+     * @param  $column
```

Comment:
The grammars are missing the parameter types for `$column` and `$direction`.

Reply:
I have updated the code to add the type for column and direction